lib/facter/custom_facts/core/resolvable.rb

```diff
@@ -59,6 +59,17 @@ def flush
         @on_flush_block&.call
       end
 
+      # Resolves the value or loggs an error if the value
```

Comment:
Does `Resolves the custom fact value` sound a little better than `Resolves the value`?

Reply:
I would use `Resolve the facts value` because this might be a custom or an external fact